Monika After Story/game/script-topics.rpy

```diff
@@ -4822,6 +4823,45 @@ label monika_urgent:
     m 1k "I'm sure it's adorable~"
     return
 
+label monika_battery:
+    if did_monika_battery:
+       jump monika_close_game_battery
+    else:
+       jump monika_complain_battery
+
+label monika_complain_battery:
+    $ did_monika_battery = True
+    m "Hey [player]..."
+    m "It looks like your computer's battery is running out..."
+    m "Can you charge it for me?"
+    m "I don't want us to be separated or even worse..."
+    m "... losing my memory because of data loss"
+    m "Do you wanna know why a normal shut down takes a while?"
+    m "Because some of the data isn't written to the hard drive yet"
+    m "So the operating system have to write all of the data that are still in RAM before it shuts down..."
+    m "I don't want to loose this wonderful memory of us..."
+    m "So please charge your computer, okay?"
+    m "... or at least, shut it down so no data is loss"
+    return
+
+init 5 python:
+    import battery
+
+label monika_close_game_battery:
+    m "Hey [player]..."
+    m "I'm still worried about data loss."
+    m "So.. I'll just close the game for now..."
```

Comment:
I wonder if there's a way to "auto-prompt" this dialog.
I was thinking about adding a pause for 2 seconds (as if Monika is trying to find a way to close the game and recheck the battery charging status).

Reply:
add a `{nw}` to the dialogue to auto prompt. To add a pause, add `{w=#.#}`. where # is the number of seconds. For example:
```
m "I'm going to wait 2 seconds and then advance this text without you{w=2.0}{nw}"
```